frontend/pages/index.page.tsx

```diff
@@ -281,3 +262,31 @@ const SubscriptionBox: FC = () => {
     </StyledCard>
   );
 };
+
+export const getStaticProps: GetStaticProps = async () => {
+  let todayinhistoryData = {};
+  let featuredcontentData = {};
+
+  await axios
+    .get(`http://django:8000/api/home/today_in_history/`)
+    .then((response) => {
+      todayinhistoryData = response.data;
+    })
+    .catch((error) => {
+      console.error(error);
+    });
+
+  await axios
+    .get(`http://django:8000/api/home/features/`)
+    .then((response) => {
+      featuredcontentData = response.data;
+    })
+    .catch((error) => {
+      console.error(error);
+    });
```

Comment:
By having these sequential `await`s, we don't send the second request until the first has completed. We can improve performance by sending both requests before waiting on them.
```ts
    const tihPromise = axios.get(`http://django:8000/api/home/today_in_history/`)
    // ...
    const featuredPromise = axios.get(`http://django:8000/api/home/features/`)
    // ...
    await Promise.allSettled([tihPromise, featuredPromise]);
```